src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -43,13 +43,13 @@ try
         return DefWindowProc(hwnd, WM_NCCREATE, wParam, lParam);
     }
 #pragma warning(suppress : 26490) // Win32 APIs can only store void*, have to use reinterpret_cast
-    auto terminal = reinterpret_cast<HwndTerminal*>(GetWindowLongPtr(hwnd, GWLP_USERDATA));
+    const auto publicTerminal = reinterpret_cast<HwndTerminal*>(GetWindowLongPtr(hwnd, GWLP_USERDATA));

```

Comment:
I renamed this variable so that it is named identical to all the other code in this file. This helped me review my changeset and to use Ctrl+F & replace.